components/lib/crash/src/main/java/mozilla/components/lib/crash/service/CrashReporterService.kt

```diff
@@ -19,4 +20,9 @@ interface CrashReporterService {
      * Submits a crash report for this [Crash.NativeCodeCrash].
      */
     fun report(crash: Crash.NativeCodeCrash)
+
+    /**
+     * Record breadcrumb with this service
+     */
+    fun recordCrashBreadcrumb(crashBreadcrumb: CrashBreadcrumb) { /* default implementation */ }
```

Comment:
Decided the add the default implementation so override is just optional.  Is this good or should I make it a mandatory override for all crash reporter services.

Reply:
I think that's a great idea! In Kotlin you can just write `= Unit` to have an empty default implementation. :)

Reply:
Done.